embedx/config.schema.json

```diff
@@ -1564,6 +1564,13 @@
                       "default": 8,
                       "minimum": 6
                     },
+                    "max_password_length": {
+                      "title": "Maximum Password Length",
+                      "description": "Defines the maximum length of the password.",
+                      "type": "integer",
+                      "default": 1024,
+                      "minimum": 20
```

Comment:
Note: Defaults should be decided by upstream

Reply:
question(non-blocking): It's in bytes, isn't it?